lib/src/annotation/point_annotation_manager.dart

```diff
@@ -7,6 +7,9 @@ class PointAnnotationManager extends BaseAnnotationManager {
       {required String id, required BinaryMessenger messenger})
       : super(id: id, messenger: messenger);
 
+  /// Needs for handling tap on cluster
+  String get clusterLayerId => 'mapbox-iOS-cluster-circle-layer-manager-$id';
```

Comment:
This feels like should be done in the application layer (App / example app) keep flutter sdk shoudl be platform agnostic.

Refer to iOS and flutter sdk examples, these kinds of hardcoded value are more like un-official constants that available to developer so SDK shouldn't provide them. 

But since we are trying I think we can move on with it. And maybe move to helper extension in another file with both iOS and android constants. 

And for our own customisation, we should definitely make our own life as SDK consumer easier by providing these kinds of helper value.